### ISSUE TITLE
Bookstore: Use Redux in React components.

### ISSUE THREAD

**Issue by Marcraphael12** [Marcraphael12/react-book-store#3]
### Project requirements
- First of all - make sure that you keep state immutable in reducers!
- Import Redux `Provider` and your store in the main component of your React app.
- Connect your app to the Redux store.
- Make sure that list of books displays books from Redux store.
    - Use `useSelector` to consume the state.
- Make sure that categories page displays categories from Redux store.
    - Use `useSelector` to consume the state.
- In your React component responsible for adding new books, add funcionality for "Add new" button click:
     - Import Redux `useDispatch`.
     - Set the data inputs in the local React state (set *title* and *author*  - NOTE: *categories* and *comments* are NOT part of this step). 
     - Once your new book object is ready to be submitted to Redux store, dispatch a corresponding action (import Action Creator that you created before and use it here).
- In your React component responsible for removing books, add a funcionality for "Remove" button click:
    - Import Redux `useDispatch`.
    - Dispatch a corresponding action (import Action Creator that you created before and use it here).
- In your React component responsible for categories, add a funcionality for "Check status" button click:
    - Import Redux `useDispatch`.
    - Dispatch a corresponding action (import Action Creator that you created before and use it here).
